tests/fuzz/README.md

```diff
@@ -0,0 +1,97 @@
+# fuzzer
+
+A quick little fuzzer for detecting issues within each of the five context areas.
+
+# How it works
+
+The fuzzer creates testcases by mutating entries within [bin/seed.xss](bin/seed.xss). The entries are then stored into redis as:
```

Comment:
I just scraped from a bunch of different lists (via Googling) and modified them a tiny bit.